content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
+Banyak user-user React yang membaca kredit [Thinking in React](/docs/thinking-in-react.html) sebagai momen dimana React akhirnya "clicked" untuk mereka. Ini mungkin panduan React tertua tetapi masih relevan.
 
-### Recommended Courses {#recommended-courses}
+### Rekomendasi Kursus {#recommended-courses}
 
-Sometimes people find third-party books and video courses more helpful than the official documentation. We maintain [a list of commonly recommended resources](/community/courses.html), some of which are free.
+Terkadang orang mencari buku dan video kursus lebih membantu daripada dokumentasi resmi. Kami memelihara [daftar sumber daya rekomendasi yang umum](/community/courses.html), beberapa di antaranya gratis.
 
-### Advanced Concepts {#advanced-concepts}
+### Konsep Lanjutan {#advanced-concepts}
 
-Once you're comfortable with the [main concepts](/docs/hello-world.html) and played with React a little bit, you might be interested in more advanced topics. This section will introduce you to the powerful, but less commonly used React features like [context](/docs/context.html) and [refs](/docs/refs-and-the-dom.html).
+Setelah Anda merasa nyaman dengan [konsep utama](/docs/hello-world.html) dan bermain dengan React, Anda mungkin tertarik dengan topik yang lebih lanjut. Bagian ini akan memperkenalkan Anda pada fitur React yang *powerful*, tetapi kurang umum digunakan seperti [context](/docs/context.html) and [refs](/docs/refs-and-the-dom.html).
 
-### API Reference {#api-reference}
+### Referensi API {#api-reference}
 
-This documentation section is useful when you want to learn more details about a particular React API. For example, [`React.Component` API reference](/docs/react-component.html) can provide you with details on how `setState()` works, and what different lifecycle methods are useful for.
+Bagian dokumentasi ini berguna ketika anda ingin belajar lebih detail mengenai *React API*.  Bagian dokumentasi ini berguna ketika Anda ingin belajar lebih jauh mengenai React API. Misalnya, [referensi API `React.Component`](/docs/react-component.html) dapat memberi anda detail dari bagaimana `setState()` bekerja, dan perbedaan serta kegunaan metode *lifecycle*.
 
-### Glossary and FAQ {#glossary-and-faq}
+### Glosarium dan FAQ {#glossary-and-faq}
 
-The [glossary](/docs/glossary.html) contains an overview of the most common terms you'll see in the React documentation. There is also a FAQ section dedicated to short questions and answers about common topics, including [making AJAX requests](/docs/faq-ajax.html), [component state](/docs/faq-state.html), and [file structure](/docs/faq-structure.html).
+[Glosari](/docs/glossary.html) berisi ikhtisar paling umum yang akan ada lihat pada dokumentasi React. Ada juga bagian *FAQ* yang didedikasikan pertanyaan dan jawaban singkat tentang topik umum, termasuk [membuat AJAX requests](/docs/faq-ajax.html), [component state](/docs/faq-state.html), dan [file structure](/docs/faq-structure.html).
```

Comment:
This `glossary` translation also inconsistent. On the heading you use `Glosarium` while in here you use `Glosari`. Both should be `Glosarium`